tf-modules/ingest/outputs.tf

```diff
@@ -50,6 +50,12 @@ output "kinesis_inbound_event_logger_lambda_function_arn" {
   value = aws_lambda_function.kinesis_inbound_event_logger.arn
 }
 
+output "lzards_backup_lambda_function_arn" {
+  value = length(aws_lambda_function.lzards_backup_task) > 0
+    ? ({ task_arn = aws_lambda_function.lzards_backup_task[0].arn })
+    : { task_arn = ""}
```

Comment:
Is returning an object with a `task_arn` property different from the rest of our outputs, where the output itself is a direct reference to the task ARN?

Reply:
I don't believe this is functionally different than 

```
output "move_granules_task" {
  value = {
    task_arn = aws_lambda_function.move_granules_task.arn
  }
}
```

Functionally in output both look like: 

lzards_backup_task = {
  "task_arn" = "arn:aws:lambda:us-east-1:SNIP:function:jk-tf4-LzardsBackup"
}


aside from compensating for the case where the switch might result in no lambda deployed.    Functionally this means that the output could be either the ARN *or* "", depending on your configuration. 



Annoyingly the multi-line syntax didn't actually work.

Reply:
5713bd57c -- I didn't deploy when I did the quick readability refactor and missed the enclosing parens.

Reply:
OK. One thing I do notice is that for other workflow tasks, we name the output differently, like so

```
output "hyrax_metadata_updates_task" {
  value = {
    task_arn = aws_lambda_function.hyrax_metadata_updates_task.arn
    task_log_group = aws_cloudwatch_log_group.hyrax_metadata_updates_task.name
  }
}
```

whereas this output is named `lzards_backup_lambda_function_arn` but really contains an object with the task_arn property. Definitely a nit, but possibly worth changing

Reply:
Agreed, it's being renamed in tf-modules/cumulus:

```
output "lzards_backup_task" {
  value = module.ingest.lzards_backup_lambda_function_arn
}
```

364430591